src/account/account.repository.knex.ts

```diff
@@ -42,4 +42,48 @@ export class KnexAccountRepository {
             site,
         );
     }
+
+    async getByApId(apId: URL): Promise<Account | null> {
```

Comment:
🤔 would this be more appropriate at the service level? Feels weird creating an object literal of a `Site` within the method (and leaks `site` implementation details into this repository)

Reply:
Hm, i see we already have this method at this level. Maybe we pass the site in?

Reply:
It does leak Site a bit - but I think `Site` is intrinsically tied to `Account`, because an `Account` is internal or external based on a relation with a `Site` through the `users` table. If anything I would move `Site` closer into account here 🤔